support/linux/install_dev_9_linux.sh

```diff
@@ -43,6 +43,6 @@ else
   sudo -E addgroup --system hab || true
 fi
 
-sudo sh /tmp/install.sh
+sudo bash /tmp/install.sh
```

Comment:
Yep, totally, not sure how this was included... (I probably branched off #6113 instead of master)

Is there an easy/preferred way to revert this change?  (I can always remote the two characters and commit the change)

Reply:
> Yep, totally, not sure how this was included... (I probably branched off #6113 instead of master)
> 
> Is there an easy/preferred way to revert this change? (I can always remote the two characters and commit the change)

Another commit on top of this that reverts it is fine. I don't think this PR should have any net changes to this file when it merges. As long as that's the case, I'm happy.

Reply:
:+1: